README.md

```diff
@@ -7,25 +7,60 @@ A [MIRA](https://github.com/TuftsUniversity/MIRA) replacement running on [Hyrax]
 
 ## Development
 
-Get started by cloning the repository and installing the dependencies:
+### dependencies
+* MySQL
+* Redis
+* Path to [FITS](https://projects.iq.harvard.edu/fits/downloads) in the [Hyrax initializer](https://github.com/curationexperts/epigaea/blob/master/config/initializers/hyrax.rb)
+* sidekiq as queue adapter in [application.rb](https://github.com/curationexperts/epigaea/blob/master/config/):        `config.active_job.queue_adapter = :sidekiq`
+
+
+### initial setup
+Get started by cloning the repository, installing the dependencies, and ensuring the test suite runs:
 
 ```sh
 git clone https://github.com/curationexperts/epigaea.git
 cd epigaea
-
-cp .env.sample .env.development
 bundle install
-bundle exec rails db:setup
 bundle exec sidekiq -d -l tmp/sidekiq.log
-# open a separate session and run bundle exec rails hydra:server
+
+# start each of the next services in their own terminal session
+bundle exec rake hydra:server 
+bundle exec rake hydra:test_server
+
+#back in your main terminal session
+bundle exec rails db:setup
+bundle exec rake spec
 ```
-Other services and settings required:
-* MySQL
-* Redis
-* Path to [FITS](https://projects.iq.harvard.edu/fits/downloads) in the [Hyrax initializer](https://github.com/curationexperts/epigaea/blob/master/config/initializers/hyrax.rb)
-* sidekiq as queue adapter in [application.rb](https://github.com/curationexperts/epigaea/blob/master/config/):        `config.active_job.queue_adapter = :sidekiq`
 
-You can run CI with `rake` (or `rake ci`). Or start a server with `rake hydra:server`
+### running the development environment server
+You need to start up the LDAP server using the `ladle` task and Solr, Fedora, and a webserver using the `hydra:server` task.
+It's usually best to run eadh service in it's own terminal session.
+```sh
+# if you checked out new code, run the next two commands
+# bundle install
+# bundle exec rake db:migrate
+bundle exec rake ladle      #start an LDAP server in a new window
+bundle exec hydra:server    #start the development server, fedora, and solr in a new window
+# visit http://localhost:3000
+```
+
+The applicaiton is configured to use LDAP for authentication.  The development and test 
+environments use the [ladle](https://github.com/NUBIC/ladle) gem to launch a self-contained LDAP server.
+LDAP users are seeded from the file at `config/ldap_seed_users.ldif`, so you can login
+using either `user@example.org` or `admin@example.org` with the passord 'password'.
+
+### making an admin user
+First, you'll need to start your development server and login as one of the LDAP users.  
+We'll assume you logged in as `admin@example.org`
+```sh
+bundle exec rails c
+> u = User.find_by(email: 'admin@example.org')
+> u.add_role('admin')
+> exit
```

Comment:
I think we need `u.save!` here?

Reply:
The `add_role` method should be taking care of the save.  It's actually the Roles table that's getting updated, but you need to reload the user to make the effects show.  Did you run into problems with this?  If so, we need to look into `add_role` and fix it:
https://github.com/curationexperts/epigaea/blob/master/app/models/user.rb#L34-L40

Reply:
No problems, just thought it was necessary. I'd created an admin earlier.